### DISCUSSION TITLE
Not working on M1 chips?

### DISCUSSION THREAD

**Discussion by JiazhengSun** [abiosoft/colima#355] [Q&A] [answered]
Hi, I am running macOS Monterey 12.2.1 with M1 Max. When I try to install colima through homebrew, it says:

colima dependencies not built for the arm64 CPU architecture:
  m4 was built for x86_64
  autoconf was built for x86_64

Any idea how to proceed? Thanks

**Comment by rfay** [comment, marked as answer]
You seem to be using an amd64 version of homebrew. Quickly! Uninstall your existing homebrew and install the arm64 version of homebrew. You probably have homebrew in /usr/local/bin, right? That's the wrong one. You want the one that will be installed in /opt/homebrew. Having the wrong homebrew will mess up everything you do.

**Reply by JiazhengSun** [reply]
The path is actually /usr/local/opt/brew....is there a way to check which arthitecture the homebrew is targeting?

**Reply by JiazhengSun** [reply]
nvm, the correct one should be /opt/... Thanks!

**Comment by vraravam** [comment]
(just a guess) Could it also be that homebrew was installed after turning on rosetta?

**Reply by JiazhengSun** [reply]
Yeah I think that might be why. Also following rfay's suggestion, let me try to uninstall the current one and reinstall without Resetta and see if it works.

**Comment by JiazhengSun** [comment]
Thank you so much for the answer! Yes the issue was indeed installing after opening terminal with Rosetta. Now it works like a charm!

**Reply by rfay** [reply]
Whatever you do, do not use rosetta for homebrew on an M1 machine. Homebrew has had native arm64 binaries since just a couple of months after the M1 came out. You will have unpredictable and sad results if you use rosetta to run amd64 binaries. Reinstall homebrew properly. It will end up in /opt.
